Domain/Entities/Item.cs

```diff
@@ -1,9 +1,17 @@
-﻿namespace Domain.Entities
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Domain.Entities
 {
     public class Item
     {
-        public int Id { get; set; }
+        [DatabaseGenerated(DatabaseGeneratedOption.None)]
+        [Key]
+        public long Id { get; set; }
         public string Name { get; set; }
         public string Description { get; set; }
+        public List<String> ImageUrlList { get; set; }
```

Comment:
change to ImageUrls, the word List is redundant because the type already specifies it